charts/uppmax-integration/templates/service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "uppmax-integration.name" . }}
+spec:
+  selector:
+    app: {{ include "uppmax-integration.name" . }}
+  ports:
+    - name: http
+      port: {{ .Values.service.port }}
```

Comment:
Maybe this should be a ternary based on if the application is started with TLS enabled or not.
Currently the app is not using TLS but it will in the future.